kolibri/core/assets/src/views/core-modal.vue

```diff
@@ -131,6 +131,12 @@
         if (!this.$refs.modal) {
           return;
         }
+        if (event.target === this.$refs.modal) {
+          return;
+        }
+        if (this.$refs.modal.contains(event.target.activeElement)) {
+          return;
+        }
```

Comment:
could you add comments for each of those exceptions explaining when they occur, similar to the other examples?

Reply:
clarified here: https://github.com/learningequality/kolibri/pull/3857/commits/f703a7795c2e88a55bc5583e396428aeac435e4d